server/src/services/metadata.service.spec.ts

```diff
@@ -149,15 +155,19 @@ describe(MetadataService.name, () => {
         id: assetStub.image.id,
         duration: null,
         fileCreatedAt: fileModifiedAt,
+        fileModifiedAt,
         localDateTime: fileModifiedAt,
       });
     });
 
     it('should take the file creation date when missing exif and earliest than modification date', async () => {
```

Comment:
Not from your PR but would you mind changing all these "earliest" to "earlier" while you're here?